src/LanguageServer/ProtocolUnitTests/Metadata/LspMetadataAsSourceWorkspaceTests.cs

```diff
@@ -21,7 +22,7 @@ public LspMetadataAsSourceWorkspaceTests(ITestOutputHelper testOutputHelper) : b
     }
 
     [Theory, CombinatorialData]
-    public async Task TestMetadataFile_OpenClosed(bool mutatingLspWorkspace)
+    public async Task TestMetadataFile_OpenClosed(bool mutatingLspWorkspace, bool useVirtualFiles)
```

Comment:
lovely.  did you update all tests to have this flag?

Reply:
Yeah I updated pretty much all the metadata tests to have this flag (except specific tests that were testing irrelevant parts of the subsystem).  See also the test changes in EditorFeatures

Reply:
This is ensuring that we'll still work over LSP even if the client doesn't support virtual files? Which I guess would matter if go to def or something moved in VS?